### PR TITLE
Update contributing guidelines

### PR DESCRIPTION
**Pull request by tundal45**
Minor formatting changes and additions to improve clarity.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,27 +2,25 @@
...
-[code of conduct]: https://github.com/rubyforgood/code-of-conduct
-
-Here are the basic steps to submit a pull request. Make sure that you're working on an [open issue]–if the relevant issue doesn't exist, open it!
+Here are the basic steps to submit a pull request.
 
-[open issue]: https://github.com/rubyforgood/casa/issues
+1. Claim an issue on [our issue tracker][issues] by assigning it to yourself (core team member) or commenting. If the issue doesn't exist yet, open it.
 
-1. Claim an issue on [our issue tracker][open issue] by assigning it to yourself (core team member) or commenting. If the issue doesn't exist yet, open it.
+2. Fork the [repo] and clone your forked repo locally on your machine.
```
Comment (compwron) [inline CONTRIBUTING.md:11]: If you change all the 2. 3. 4. to `1.` then it will auto-increment for you which is more convenient

**Comment by compwron** [comment]
I'd like to see a mention of the slack channel and how to join it
...how *do* you join it?

File: CONTRIBUTING.md (hunk 1 above)
Comment (tundal45) [reply CONTRIBUTING.md]: Lol an ex co-worker used to do that and I always wondered why until I was re-numbering them just now but was so committed to my old ways that did not thing to change. Updated.

**Comment by tundal45** [comment]
> I'd like to see a mention of the slack channel and how to join it
> ...how _do_ you join it?

I thought about whether that step is necessary, I can include it but figured it might be too many steps!

**Pull request merged**